PowerPointLabs/Test/FunctionalTest/HighlightBulletsTest.cs

```diff
@@ -16,6 +16,9 @@ protected override string GetTestingSlideName()
         public void FT_HighlightBulletsTest()
         {
             // Do tests in reverse order because added slides change slide numbers lower down.
+            TestRemoveHighlighting_HighlightText();
```

Comment:
one of the benefits of using Action framework is that,
now u can write the new test in UT (much faster than FT). but up to u whether to rewrite the test